bundle/regal/rules/bugs/top-level-iteration/top_level_iteration.rego

```diff
@@ -17,7 +17,7 @@ report contains violation if {
 		part.type == "var"
 	]) == 0
 
-	some part in array.slice(rule.head.value.value, 1, 128)
+	some part in array.slice(rule.head.value.value, 1, 100)
```

Comment:
would a constant somewhere be worthwhile here? so we can explain why this value was chosen?

Reply:
This is a micro-optimization, and using a constant would mean we'd need to query the virtual cache in each request, which while reasonably fast is still a cost comparable to calling `count` 😄

Reply:
(I totally agree in principle though!)

Reply:
Fair enough! 🙂